deploy/basic/terraform/schema.yaml

```diff
@@ -44,14 +48,19 @@ variables:
     required: true
     title: "Database Name"
     description: "The name for the Autonomous Database instance"
-    default: "mushop"
+    default: "MuShopDB"
```

Comment:
nit: its convention to have the db name all lowercase.

Reply:
Default creation is all UPPERCASE:
<img width="544" alt="image" src="https://user-images.githubusercontent.com/551007/83807869-5c6b2200-a679-11ea-8e11-d68d05fce940.png">

Solution Hub chose to use `AlwaysFree` on the name.